roles/lib_openshift/library/oc_adm_router.py

```diff
@@ -2665,7 +2668,7 @@ def _prepare_router(self):
         results = self.openshift_cmd(cmd, oadm=True, output=True, output_type='json')
 
         # pylint: disable=no-member
```

Comment:
Is this pylint disable still needed? If so, would disabling `maybe-no-member` work instead?